baybe/searchspace/continuous.py

```diff
@@ -211,11 +210,12 @@ def param_names(self) -> tuple[str, ...]:
         return tuple(p.name for p in self.parameters)
 
     @property
-    def param_bounds_comp(self) -> np.ndarray:
-        """Return bounds as numpy array."""
-        if not self.parameters:
-            return np.empty((2, 0), dtype=DTypeFloatNumpy)
-        return np.stack([p.bounds.to_ndarray() for p in self.parameters]).T
+    def comp_rep_bounds(self) -> pd.DataFrame:
+        """The minimum and maximum values of the computational representation."""
+        pd.DataFrame(
```

Comment:
Is no `return` needed here? Also, since the fallback `if not self.parameters` is gone, has been tested/verified what happens if this method is called in such a situation resp. is it guaranteed that this cannot happen somewhere else?

Reply:
Regarding the missing return: I'm asking in particular since the corresponding discrete subspace actually has a `return`

Reply:
Noticed the missing return myself while working on it -> has been fixed in the meantime (stuff was also not yet mypy-ed at that time, which would have shown it). And yes, has been tested for empty spaces, both in the conti and the discrete case. Returns a corresponding empty dataframe with correct min-max index 👍🏼

Reply:
There is even a test in our test module that checks it. I've now also updated it so that the dataframe indices are checked as well